src/Model/Package.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use BringYourOwnIdeas\Maintenance\Tasks\UpdatePackageInfo;
+
+/**
+ * Describes an installed composer package version.
+ */
+class Package extends DataObject
+{
+    private static $db = [
+        'Name' => 'Varchar(255)',
+        'Description' => 'Varchar(255)',
+        'Version' => 'Varchar(255)',
+        "Type" => 'Varchar(255)',
```

Comment:
Any reason for the mix of quote types?

Reply:
None, a side effect of me copying existing values for new keys I think. Can fix.